### PR TITLE
feat: Enhance MIME Type Detection for Additional File Extensions

### PR DESCRIPTION
**Pull request by gdy666**
By the way, before enhancing MIME type detection for additional file extensions, it's surprising that even the MIME type for .flac files cannot be recognized in the Linux arm64 environment.

音乐类型

QQ截图20240501194015

### CODE REVIEW

**Comment by baur** [comment]
can you fix this #2603 also 

image

**Comment by gdy666** [comment]
> can you fix this #2603 also
> 
> image

xlsx Already included

**Comment by gdy666** [comment]
> can you fix this #2603 also
> 
> image



> can you fix this #2603 also
> 
> image

The problem in your screenshot has nothing to do with the problem involved in this PR. Your problem involves front-end css style processing. Let me think about whether there is a good way to deal with it for you.

**Comment by gdy666** [comment]
> can you fix this #2603 also
> 
> image

https://github.com/filebrowser/filebrowser/pull/3187

**Comment by gdy666** [comment]
> can you fix this #2603 also
> 
> image
@baur 
You can refer to this
https://github.com/filebrowser/filebrowser/pull/3187

File: files/mime.go [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+	".sldx":      "application/vnd.openxmlformats-officedocument.presentationml.slide",
+	".sldm":      "application/vnd.ms-powerpoint.slide.macroEnabled.12",
+	".thmx":      "application/vnd.ms-officetheme",
+	".onetoc":    "application/onenote",
+	".onetoc2":   "application/onenote",
+	".onetmp":    "application/onenote",
+	".onepkg":    "application/onenote",
+	".xpi":       "application/x-xpinstall",
+	".wasm":      "application/wasm",
+	//---
```
Comment (o1egl) [inline files/mime.go]: What is that for?

File: files/mime.go [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+	".opus": "video/ogg",
+	".m4v":  "video/mp4",
+	".mkv":  "video/x-matroska",
+	".caf":  "audio/x-caf",
+	".m3u8": "application/x-mpegURL",
+	".mpd":  "application/dash+xml",
+	".webp": "image/webp",
+}
+
+func init() {
```
Comment (o1egl) [inline files/mime.go:603]: Add nolint comment to suppress linter error

File: files/mime.go (hunk 1 above)
Comment (gdy666) [reply files/mime.go]: sorry,It is only used to distinguish the part from the github.com/kataras/iris. It would be more appropriate to remove this line.

**Pull request merged**